src/components/ControlBar/ViewControlBar/styles/DragHandle.module.css

```
@@ -0,0 +1,15 @@
+.draghandle {
+    position: absolute;
+    bottom: 0px;
```

```suggestion
    bottom: 0;
```
Same for the others below ⬇️

---

Done